lib/optimizely/test.js

```diff
@@ -57,6 +62,25 @@ describe('Optimizely', function(){
           done();
         });
       });
+      it('should call #sendRoots if listen is enabled', function(done){
+        optimizely.options.listen = true;
+        analytics.initialize();
```

Comment:
All of this initialize business should be moved out into a beforeEach, like in [this test suite](https://github.com/segmentio/analytics.js-integrations-private/blob/7b1206d9a002a475bc5a0c9a01033ce9ba726b53/lib/appnexus/test.js#L59)